test/jdk/java/awt/List/ListScrollbarTest.java

```diff
@@ -0,0 +1,196 @@
+                   to move the item to the right list.
+                6. If horizontal scroll bar appears on the right list
+                   as well as disappears from the left list [only if both
+                   happen] proceed with step 8 else click FAIL
+                7. Now move the same item to the left list, by pressing
+                     '<' button
+                8. If the horizontal scrollbar appears on the left list
+                   and disappears from the right list[only if both happen]
+                   click PASS else click FAIL.
+                """;
+        PassFailJFrame.builder()
+                .title("Test Instructions")
+                .instructions(INSTRUCTIONS)
+                .columns(35)
+                .testUI(ListScrollbarTest::new)
+                .build()
+                .awaitAndCheck();
+    }
+
+    public ListScrollbarTest() {
+        super("List scroll bar test");
+        GridBagLayout gbl = new GridBagLayout();
+        ltList = new List(ITEMS, true);
+        rtList = new List(0, true);
+        setLayout(gbl);
+        add(ltList, 0, 0, 1, 5, 1.0, 1.0);
+        add(rtList, 2, 0, 1, 5, 1.0, 1.0);
+        add(new Button(">"), 1, 0, 1, 1, 0, 1.0);
+        add(new Button(">>"), 1, 1, 1, 1, 0, 1.0);
+        add(new Button("<"), 1, 2, 1, 1, 0, 1.0);
+        add(new Button("<<"), 1, 3, 1, 1, 0, 1.0);
+        add(new Button("!"), 1, 4, 1, 1, 0, 1.0);
+
+        for (int i = 0; i < ITEMS; i++) {
+            ltList.addItem("item " + i);
+        }
+        setSize(220, 250);
+    }
+
+    void add(Component comp, int x, int y, int w, int h, double weightx, double weighty) {
+        GridBagLayout gbl = (GridBagLayout) getLayout();
+        GridBagConstraints c = new GridBagConstraints();
+        c.fill = GridBagConstraints.BOTH;
+        c.gridx = x;
+        c.gridy = y;
+        c.gridwidth = w;
+        c.gridheight = h;
+        c.weightx = weightx;
+        c.weighty = weighty;
+        add(comp);
+        gbl.setConstraints(comp, c);
+    }
+
+    void reverseSelections(List l) {
+        for (int i = 0; i < l.countItems(); i++) {
+            if (l.isSelected(i)) {
+                l.deselect(i);
+            } else {
+                l.select(i);
+            }
+        }
+    }
+
+    void deselectAll(List l) {
+        for (int i = 0; i < l.countItems(); i++) {
+            l.deselect(i);
+        }
+    }
+
+    void replaceItem(List l, String item) {
+        for (int i = 0; i < l.countItems(); i++) {
+            if (l.getItem(i).equals(item)) {
+                l.replaceItem(item + "*", i);
+            }
+        }
+    }
+
+    void move(List l1, List l2, boolean all) {
+
+        // if all the items are to be moved
+        if (all) {
+            for (int i = 0; i < l1.countItems(); i++) {
+                l2.addItem(l1.getItem(i));
+            }
+            l1.delItems(0, l1.countItems() - 1);    // or l1.clear();
+        } else { //else move the selected items
```

Comment:
For consistency in comments

```suggestion
        } else { // else move the selected items
```

Reply:
Removed.